### ISSUE TITLE
ERR Failed to serve quic connection error="Unauthorized: Failed to get tunnel" 

### ISSUE THREAD

**Issue by dnedeltchev** [brenner-tobias/addon-cloudflared#494]
### The problem

I had Cloudflared running just fine on an old free domain from freenom. I decided to buy a proper domain from cloudflare and use that one. Once I moved the config to the new domain, I get the below on a loop in the logs.

2023-10-09T00:47:28Z ERR Failed to serve quic connection error="Unauthorized: Failed to get tunnel" connIndex=0 event=0 ip=198.41.192.67
2023-10-09T00:47:28Z ERR Register tunnel error from server side error="Unauthorized: Failed to get tunnel" connIndex=0 event=0 ip=198.41.192.67
2023-10-09T00:47:29Z INF Retrying connection in up to 2s connIndex=0 event=0 ip=198.41.192.67

### What version of Cloudflared has the issue?

4.2.7

### What was the last working version of Cloudflared?

4.2.7

### What type of installation are you running?

Home Assistant OS

### Add-on YAML Configuration

```yaml
external_hostname: domain.xyz
additional_hosts: []
tunnel_name: homeassistant
```

### Anything in the logs that might be useful for us?

```txt
2023-10-09T00:47:28Z ERR Failed to serve quic connection error="Unauthorized: Failed to get tunnel" connIndex=0 event=0 ip=198.41.192.67
2023-10-09T00:47:28Z ERR Register tunnel error from server side error="Unauthorized: Failed to get tunnel" connIndex=0 event=0 ip=198.41.192.67
2023-10-09T00:47:29Z INF Retrying connection in up to 2s connIndex=0 event=0 ip=198.41.192.67
```

### Steps to reproduce the issue

start the add-on

**Comment by darkenbula** [comment]
same as above after updating to latest version of addon

**Comment by dnedeltchev** [comment]
> same as above after updating to latest version of addon

I just restored an old version, 4.2.4, and still get the same error :(

**Comment by dnedeltchev** [comment]
I went back to 4.1.0 and still the same... :(

**Comment by lukepistrol** [comment]
I have the exact same problem without changing anything. It just stopped working today.

**Comment by kwankiu** [comment]
Same as well

**Comment by felipecrs** [comment]
I have absolutely no idea on what's happening, but it happened to one of my friends though.

My friend already tried uninstalling the add-on and installing again, which means that reauthenticating it doesn't seem to resolve the problem.

Maybe, who knows, Cloudflare started banning people who are streaming video (like Frigate, go2rtc, or any other cameras) through the tunnel. As stated in their terms of service.

I would advise some of you to reach out to Cloudflare's support to get more information on the problem.

As this add-on is not anything official from Cloudflare, I would recommend you just mention you were using cloudflared and it stopped working out of the blue. Providing the logs, of course.

**Comment by felipecrs** [comment]
Actually, I can see that Cloudflare's status is reporting some unavailability:

https://www.cloudflarestatus.com/

Maybe you should just wait for its resolution first. Hopefully it will resolve the problem.

**Comment by dnedeltchev** [comment]
Indeed, looks like the Tunnel service has degraded performance...
________________________________

**Comment by darkenbula** [comment]
Back up and running now

**Comment by dnedeltchev** [comment]
Yep, that'll teach me a good lesson, which is to go to bed early and don't mess around with home assistant late at night. If I'd gone to bed on time I'd never have known about this 😁
Thank you Felipe for pointing out the cloud flare issues!
________________________________
